### PR TITLE
Make R6 apis consistent

### PR DESCRIPTION
**Pull request by schloerke**
See commits for api conversions.

PR task list:
- [x] Update NEWS
- [x] Add tests
- [x] Update documentation with `devtools::document()`

### CODE REVIEW

File: R/plumber-static.R [hunk 1]
```diff
@@ -5,13 +5,13 @@
 #' @include plumber.R
 #' @export
 PlumberStatic <- R6Class(
-  "plumberstatic",
+  "PlumberStatic",
```
Comment (cpsievert) [inline R/plumber-static.R:8]: Make sure to mention this as a breaking change in the news

File: R/plumber.R [hunk 2]
```diff
@@ -77,11 +77,11 @@ Plumber <- R6Class(
...
       self$setErrorHandler(defaultErrorHandler())
       self$set404Handler(default404Handler)
-      self$setUi()
+      self$setUi(TRUE)
       private$ui_info$has_not_been_set <- TRUE # set to know if `$setUi()` has been called before `$run()`
-      self$setUiCallback()
-      self$setDebug()
-      self$set_api_spec()
+      self$setUiCallback(getOption('plumber.ui.callback', getOption('plumber.swagger.url', NULL)))
+      self$setDebug(TRUE)
```
Comment (cpsievert) [inline R/plumber.R:83]:
```suggestion
      self$setDebug(interactive())
```

File: NEWS.md [hunk 3]
```diff
@@ -20,7 +20,7 @@ plumber 1.0.0
 
 ### Breaking changes
 
-* When `plumb()`ing a file (or `plumber$new(file)`), the working directory is set to the file's directory before parsing the file. When running the Plumber API, the working directory will be set to file's directory before running.(#631)
+* When `plumb()`ing a file (or `Plumber$new(file)`), the working directory is set to the file's directory before parsing the file. When running the Plumber API, the working directory will be set to file's directory before running.(#631)
```
Comment (cpsievert) [inline NEWS.md:24]: Let's also add a comment about `plumb()` now returning an object of class `Plumber` (not `plumber`), but you can now use `is_plumber()` if you need to check the class.

**Comment by schloerke** [comment]
Minimal News items to add:

- [ ] `plumberstatic` is now `PlumberStatic`
- [ ] `plumber` is now `Plumber`. `plumber` throws a deprecated message
- [ ] `is_plumber()` - new method

File: NEWS.md [hunk 4]
```diff
@@ -20,7 +20,11 @@ plumber 1.0.0
 
 ### Breaking changes
 
-* When `plumb()`ing a file (or `plumber$new(file)`), the working directory is set to the file's directory before parsing the file. When running the Plumber API, the working directory will be set to file's directory before running.(#631)
+* `plumb()` now returns an object of class `"Plumber"` (previously `"plumber"`). To check is the return value is a Plumber router, use new method `is_plumber()`. (#653)
```
Comment (cpsievert) [inline NEWS.md:23]:
```suggestion
* `plumb()` now returns an object of class `"Plumber"` (previously `"plumber"`). To check if an object is a Plumber router, use new method `is_plumber()`. (#653)
```

**Pull request merged**